functions/main.py

```diff
@@ -0,0 +1,97 @@
+import json
+import requests
+from bs4 import BeautifulSoup
+
+def hello(data):
+    backtext = 'Hello world!\ndata:' + str(data) + '\n'
+    print(data)
+    # print(context)
+    return backtext
+
+def get_list_from_api(title):
+    print("title:" + title)
+    headers = {
+        "Connection": "keep-alive",
+        "accept": "*/*",
+        "x-app-id": "RUMORS_SITE",
+        "DNT": "1",
+        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.62 Safari/537.36",
+        "content-type": "application/json",
+        "Origin": "https://cofacts.g0v.tw",
+        "Sec-Fetch-Site": "same-site",
+        "Sec-Fetch-Mode": "cors",
+        "Sec-Fetch-Dest": "empty",
+        "Referer": "https://cofacts.g0v.tw/",
+        "Accept-Language": "zh-TW,zh;q=0.9,en;q=0.8"
+    }
+    data_data  = { 
+        'operationName': 'ListReplies', 
+        'variables': { 
+            'filter': { 
+                'moreLikeThis': { 
+                    'like': 'N/A', 
+                    'minimumShouldMatch': '0' 
+                    } 
+                }, 
+        'orderBy': [{ 
+            '_score': 'DESC' 
+            },{ 
+            'createdAt': 'DESC' 
+            }]
+        }, 
+        'query': 'query ListReplies($filter: ListReplyFilter, $orderBy: [ListReplyOrderBy], $before: String, $after: String) { ListReplies(filter: $filter, orderBy: $orderBy, before: $before, after: $after, first: 10) { edges { node {  ...ReplyItem  __typename } cursor __typename } __typename } }  fragment ReplyItem on Reply { id text type createdAt user { id name __typename } articleReplies(status: NORMAL) { articleId replyId __typename } __typename } ' 
+    }
```

Comment:
1. I would suggest you extract headers as a const outside of this function, since other api hit would also use it
2. `data_data` is not a good variable name, and please extract make a `function` to return query from given parameter